core/templates/components/state-editor/state-hints-editor/state-hints-editor.directive.ts

```diff
@@ -276,7 +276,7 @@ angular.module('oppia').directive('stateHintsEditor', [
             $scope.canEdit = EditabilityService.isEditable();
 
             $scope.dragDotsImgUrl = UrlInterpolationService.getStaticImageUrl(
-              '/general/drag_dots.png');
+              '/general/drag_dots.webp');
```

Comment:
Could we support both PNG and WebP here?

Reply:
I have made them as picture tags in the directive.html file

Reply:
No, there are no changes to `state-hints-editor.directive.html`, also why do we need this variable if you set it up in the HTML file.

Reply:
Has this been resolved?

Reply:
done!